lib/LanguagePicker.js

```diff
@@ -1,16 +1,39 @@
+const ls = require('language-scripts');
+const dataOverrides = require('./overrides.json');
 
 function LanguagePicker(lang = 'en', config = {}) {
   let fallbackMap = {};
   let fallbacks = [];
+  const scripts = ls.adjust({ override: dataOverrides });
 
+  this.userLang = lang;
+  this.values = {};
   this.nameTag = config.nameTag;
   // The prefix is either given or is the nameTag
   // with an underscore
   // See Babel.js#24
   this.prefix = config.multiTag || (this.nameTag && `${this.nameTag}_`) || '';
-  fallbackMap = config.languageMap || {};
+
+
+  // Store language script
+  this.langScript = scripts[lang] || 'Latn';
```

Comment:
also, forgot to mention - prepend langScript with a `-` -- this would be more accurate.

Reply:
P.S. I meant to add it after you collect languages below - so that when you do `endsWith()`, it works more accurately.  Just a nitpick really.

Reply:
But even in the list you sent, there's a ja_Latn (https://taginfo.openstreetmap.org/search?q=Latn)

Should I still force it to look for -[script] ? Can we trust that structure?

Reply:
There is no `ja_Latn`, check again :)  I already fixed those a few days ago, but they usually take ~24hrs to update taginfo. We don't want to handle one-off mistakes - people are very creative in how they mess up, we don't want to encourage it :)  Someone always fixes it, sooner or later.  So yes, always check `-[script]`, where script is "Latn", "Cyrl", etc, but keep in mind that we can treat `"_rm"` for a some (all?) languages as a sinonym to `"-Latn"`.

Reply:
... I could have sworn I saw one... my eyes are deceiving me.

Okay, if we can (for the most part) trust the structure, I'll add that to the "endsWith". Thanks!

Reply:
I thought I was hallucinating... :smiley: